crates/datasources/Cargo.toml

```diff
@@ -45,7 +45,7 @@ reqwest = { workspace = true }
 rust_decimal = { version = "1.34.2", features = ["db-tokio-postgres"] }
 serde = { workspace = true }
 serde_with = "3.6.0"
-serde_json = { workspace = true }
+serde_json.workspace = true
```

Comment:
the other syntax seems to be prevalent in this file.

Reply:
seems like we have a mix of both. The latter is what's used by cargo. 

`cargo install -p datasources serde_json`